docker_deploy/README.md

```diff
@@ -0,0 +1,26 @@
+# Deploy on Kubernetes using Docker containers
+
+In our environment it was `PROJECT_DIR=$HOME/git-repos/osv-microservice-demo`.
+
+## UI
+
+Build container using the provided Dockerfile-ui:
+```bash
+$ cd $PROJECT_DIR
+$ npm install
+$ docker build -t mike/osv-microservice-demo . -f docker_deploy/Dockerfile-ui
+```
+
+Inject container to both DIND nodes since we don't know where Deployment will be scheduled:
```

Comment:
Why "both"? Should there always be two? I guess there can also be more, so replace "to both" with "to all"